app/grandchallenge/cases/views.py

```diff
@@ -146,3 +147,23 @@ class RawImageFileViewSet(
     queryset = RawImageFile.objects.all()
     permission_classes = [DjangoObjectOnlyWithCustomPostPermissions]
     filter_backends = [ObjectPermissionsFilter]
+
+    def get_permissions(self):
+        if self.action == "download":
+            return [IsAdminUser()]
```

Comment:
Wouldn't this bypass the object permissions for staff members?

Reply:
It would, but I felt like we might want to make the downloads available to all staff members, regardless of object permissions? But maybe we should check for superuser status instead?

Reply:
We shouldn't let staff members do this. I think remove the check, super users have permission anyway.

Reply:
I moved all the download logic to the admin, so this is no longer an issue